Source/WebCore/platform/graphics/egl/GLContextEGL.cpp

```diff
@@ -328,6 +330,15 @@ std::unique_ptr<GLContextEGL> GLContextEGL::createContext(GLNativeWindowType win
     }
 
     EGLContext eglSharingContext = platformDisplay.sharingGLContext() ? static_cast<GLContextEGL*>(platformDisplay.sharingGLContext())->m_context : EGL_NO_CONTEXT;
+    if (platformDisplay.type() == PlatformDisplay::Type::Headless) {
+        auto context = createPbufferContext(platformDisplay, eglSharingContext);
+        if (!context) {
+            RELEASE_LOG_INFO(Compositing, "Could not create Pbuffer context: %s.", lastErrorString());
+            WTFLogAlways("Could not create EGL context.");
```

Comment:
Why not log the error message with WTFLogAlways? The release log stuff is useful for developers but users aren't going to provide it in bug reports and we probably won't remember to ask for it. More info in stderr is helpful IMO.